dali/test/python/test_operator_pad.py

```diff
@@ -70,12 +79,14 @@ def check_pad_synth_data(device, batch_size, input_max_shape, axes):
                     assert(output_shape[dim] == max_shape[dim])
 
 def test_slice_synth_data_vs_numpy():
-    for device in ["gpu"]:
-        for batch_size in {1, 8, 100}:
-            for input_max_shape, axes in \
-                [((200,400,3), (0,)),
-                 ((200,400,3), (1,)),
-                 ((200,400,3), (0,1)),
-                 ((200,400,3), ()),
-                 ((200,400,3), [])]:
-                yield check_pad_synth_data, device, batch_size, input_max_shape, axes
+    for device in ["cpu"]:
+        for batch_size in {1, 8}:
+            for input_max_shape, axes, align in \
+                [((200, 400, 3), (0,), None),
+                 ((200, 400, 3), (1,), None),
+                 ((200, 400, 3), (0, 1), None),
+                 ((200, 400, 3), (), None),
+                 ((200, 400, 3), [], None),
+                 ((200, 400, 3), (2,), (4,)),]:
```

Comment:
Can we test other alignments as well?

Reply:
There are two cases:
```
                 ((200, 400, 3), (2,), (4,)),]:
                 ((200, 400, 3), (0, 1), (256, 256))]:
```
anything specific that you have in mind?

Reply:
added two more cases